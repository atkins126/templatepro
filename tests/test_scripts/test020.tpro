DataSets
{{if customers}}
{{for cust in customers}}
Code: {{:cust.Code}} - Name: {{:cust.Name}}
{{endfor}}
{{endif}}

List of objects
{{for obj in objects}}
Prop1: {{:obj.Prop1}} {{:value2}}
---
{{endfor}}


{{for cust in customers}}
{{:cust.@@index}}) Code: {{:cust.Code}} - Name: {{:cust.Name}}
{{for obj in objects}}
    - Prop1: {{:obj.Prop1}}
{{endfor}}
{{endfor}}
