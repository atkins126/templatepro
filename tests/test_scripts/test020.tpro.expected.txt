DataSets
Code: 1 - Name: Ford
Code: 2 - Name: Ferrari
Code: 3 - Name: Lotus
Code: 4 - Name: FCA
Code: 5 - Name: Hyundai
Code: 6 - Name: De Tomaso
Code: 7 - Name: Dodge
Code: 8 - Name: Tesla
Code: 9 - Name: Kia
Code: 10 - Name: Tata
Code: 11 - Name: Volkswagen
Code: 12 - Name: Audi
Code: 13 - Name: Skoda

List of objects
Prop1: value1.1 DANIELE2
---
Prop1: value1.2 DANIELE2
---
Prop1: value1.3 DANIELE2
---


1) Code: 1 - Name: Ford
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
2) Code: 2 - Name: Ferrari
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
3) Code: 3 - Name: Lotus
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
4) Code: 4 - Name: FCA
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
5) Code: 5 - Name: Hyundai
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
6) Code: 6 - Name: De Tomaso
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
7) Code: 7 - Name: Dodge
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
8) Code: 8 - Name: Tesla
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
9) Code: 9 - Name: Kia
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
10) Code: 10 - Name: Tata
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
11) Code: 11 - Name: Volkswagen
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
12) Code: 12 - Name: Audi
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
13) Code: 13 - Name: Skoda
    - Prop1: value1.1
    - Prop1: value1.2
    - Prop1: value1.3
